distribution/docs/src/main/resources/content/_managing/_configuring/_sources/wfs-20-source.adoc

```diff
@@ -67,18 +67,17 @@ Custom mappings of feature property responses to metacard attributes can be achi
 The `MetacardMapper` is set by creating a feature file configuration which specifies the appropriate mapping. The mappings are specific to a given feature type.
 
 .Installing the WFS `MetacardMapper`
-The WFS `MetacardMapper` is is not installed by default with a standard application in the ${ddf-spatial} application.
+The WFS `MetacardMapper` is installed by default with a standard installation in the ${ddf-spatial} application.
 
 .Configuring the WFS `MetacardMapper`
-There are two ways to configure the `MetcardMapper`, one is to use the Configuration Admin available via the ${admin-console}.
+There are two ways to configure the `MetacardMapper`, one is to use the Configuration Admin available via the ${admin-console}.
```

Comment:
✏️ This sentence is a comma splice. It might work better as:

There are two ways to configure the `MetacardMapper`:

. The Configuration Admin available in the ${admin-console}.
. Placing a `feature.xml` file in the `deploy` directory.